openstack/identity/identity.go

```diff
@@ -209,6 +212,7 @@ func (h Handler) checkToken(r *http.Request, tenant string, tokenGetResult getRe
 	/* TODO Caching or PKI */
 	for _, a := range h.ValidAdmins {
 		if validateProjectRole(tokenGetResult, a.Project, a.Role) == true {
+			context.Set(r, service.PrivKey, "true")
 			return true
```

Comment:
Is here where we can add the tenantID information to the context? it's going to be useful for ``ciao-image`` service which requires tenantID for doing the images tenant/project separation.